### PR TITLE
DDP-5925 cmi: admin clients and prism role

### PR DESCRIPTION
**Pull request by yufengwng**
## Context

Auth0 configuration changes to the CMI tenant to enable prism admin logins. This is the minimum to get this working for Angio.

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.

## FUD Score

- [x] :relaxed: All good, business as usual!

## How do we demo these changes?

- [x] I have already deployed these to `dev`

## Testing

- [x] I have written zero automated tests but have poked around locally to verify proper functionality

## Release

A few things to do:

1. Deploy the config changes using Auth0 Deploy CLI.
    ```
    $ a0deploy deploy -c output-config/cmi-config.json -i tenants/cmi/tenant.yaml
    ```
2. Find the admin clients in Auth0 dashboard, and disable all connections except google login. (I couldn't figure out a way to specify this in the `tenant.yaml` so we're doing it manually.)
3. Find the google oauth client that the CMI dev tenant is using. Go to gcloud console, find that oauth client, and add the following redirect URL:
    ```
    https://login-dev.joincountmein.org/login/callback
    ```
4. Update Vault path `secret/pepper/{env}/v1/angio/conf` and add in admin client.
5. Add new client to Pepper by running study-builder using the task `UpdateStudyClients`.

To grant prism access, we do these steps:
1. Ask them to try logging in first. They will get an error. This is needed so Auth0 creates their user profile.
2. Find their user profile in Auth0, and grant them the `CMI Prism Admin` role.
3. Take their auth0 `user_id` and add them to Pepper database, and grant access to the study.

### CODE REVIEW

File: study-builder/tenants/cmi/tenant.yaml [hunk 1]
```diff
@@ -54,6 +67,33 @@ clients:
...
+      '##ANGIO_BASE_URL##/admin-session-expired'
+      ##LOCALHOST_ADMIN_LOGOUTS##
+    ]
+    web_origins: [
+      '##ANGIO_BASE_URL##'
+      ##LOCALHOST_ORIGINS##
+    ]
+    client_metadata:
+      study: ANGIO
+      isAdminClient: 'true'
```
Comment (yufengwng) [inline study-builder/tenants/cmi/tenant.yaml:92]: I think it's possible to have a single admin client for all CMI studies. But each client has metadata about which study it is for, so I just created separate clients for each study.

**Comment by yufengwng** [comment]
Putting this in draft mode for now. There's nothing to merge yet, and I've already applied the config changes to `dev`.

**Review by MocanaAtBroad — DISMISSED** [review]
Don't see anything that appears to be wrong so :white_check_mark:

**Comment by yufengwng** [comment]
@MocanaAtBroad I think everything we need should now be in-place in this PR.

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**